reader/views.py

```diff
@@ -2692,7 +2693,9 @@ def name_api(request, name):
     LIMIT = int(request.GET.get("limit", 10))
     type = request.GET.get("type", None)
     topic_pool = request.GET.get("topic_pool", None)
-    completions_dict = get_name_completions(name, LIMIT, topic_override, type=type, topic_pool=topic_pool)
+    exact_continuations = request.GET.get("exact_continuations", False)
```

Comment:
The standard in our code (for better or worse) is to pass boolean GET parameters as `bool(int(request.GET.get("exact_continuations", False)))`